packages/core/admin/ee/server/src/controllers/audit-logs.ts

```diff
@@ -20,5 +20,7 @@ export default {
     const body = await auditLogs.findOne(id);
 
     ctx.body = body;
+
+    strapi.telemetry.send('didWatchAnAuditLog');
```

Comment:
If you want something more precise you might want to move that in the frontend so you only send the event if a user actually open a view. here you will count any api call made (might not by 1:1)

Reply:
If an API call is made here, it means the audit log is watched outside the Admin Panel, which still proves a usage of the feature, which is my goal here.